zerver/lib/event_queue.py

```diff
@@ -629,6 +679,7 @@ def receiver_is_idle(user_profile_id, realm_presences):
     return off_zulip or idle
 
 def process_message_event(event_template, users):
```

Comment:
event_template should be `Mapping[str, Any]`.
users should be `Iterable[Mapping[str, Any]]`.